neo/Network/RPC/RpcServer.cs

```diff
@@ -143,9 +146,59 @@ private JObject Process(string method, JArray _params)
         {
             switch (method)
             {
+                case "claimgas":
+                    if (Wallet == null || WalletLocker.Locked())
```

Comment:
Perhaps all these wallet related methods should be refactored into a separate file. Then they can check this condition up front instead of duplicating this check over and over in all the wallet related methods. Also it will make this file more manageable.